demos/asp-net-core/Divergent.Sales.ViewModelComposition/OrdersListViewModelAppender.cs

```diff
@@ -13,51 +14,42 @@ namespace Divergent.Sales.ViewModelComposition
 {
     public class OrdersListViewModelAppender : IViewModelAppender
     {
-        public bool Matches(RouteData routeData, string verb)
-        {
-            /*
-             * matching is a bit weak in this sample, it's designed 
-             * this way to satisfy both the composite gateway and website samples
-             */
-            var controller = (string)routeData.Values["controller"];
-
-            return HttpMethods.IsGet(verb)
-                && controller.ToLowerInvariant() == "orders"
+        // Matching is a bit weak in this demo.
+        // It's written this way to satisfy both the composite gateway and website samples.
+        public bool Matches(RouteData routeData, string verb) =>
+            HttpMethods.IsGet(verb)
+                && string.Equals((string)routeData.Values["controller"], "orders", StringComparison.OrdinalIgnoreCase)
                 && !routeData.Values.ContainsKey("id");
-        }
 
         public async Task Append(dynamic vm, RouteData routeData, IQueryCollection query)
         {
             var pageIndex = (string)query["pageindex"] ?? "0";
             var pageSize = (string)query["pageSize"] ?? "10";
 
+            // Hardcoded to simplify the demo. In a production app, a config object could be injected.
             var url = $"http://localhost:20295/api/orders?pageSize={pageSize}&pageIndex={pageIndex}";
-            var client = new HttpClient();
-            var response = await client.GetAsync(url);
+            var response = await new HttpClient().GetAsync(url);
 
             dynamic[] orders = await response.Content.AsExpandoArrayAsync();
 
-            var ordersViewModelDictionary = MapToDictionary(orders);
+            var ordersViewModel = MapToViewModel(orders);
```

Comment:
this returns a ViewModels dictionary, not a ViewModel. I'm more inclined to change the event property name from `OrdersViewModel` to `OrdersViewModelDictionary` that is what it is.

Reply:
@adamralph why this comment was ignored?

Reply:
@mauroservienti I'm working on a follow up PR right now 😉 

I merged this one because you approved it.

Reply:
OK